src/Symfony/Bridge/PhpUnit/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ CHANGELOG
  * ignore verbosity settings when the build fails because of deprecations
  * added per-group verbosity
  * added `ExpectDeprecationTrait` to be able to define an expected deprecation from inside a test
+ * deprecated `@expectedDeprecation` annotation.
```

Comment:
deprecated the `@expectedDeprecation` annotation, use the `ExpectDeprecationTrait::expectDeprecation()` method instead